sktime/performance_metrics/tests/test_metrics_classes.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+"""Tests for classes in _classes module."""
+from inspect import getmembers, isclass
+
+import numpy as np
+import pytest
+
+from sktime.performance_metrics.forecasting import _classes
+from sktime.utils._testing.series import _make_series
+
+metric_classes = getmembers(_classes, isclass)
+
+exclude_starts_with = ("_", "Base")
+metric_classes = [x for x in metric_classes if not x[0].startswith(exclude_starts_with)]
+
+names, metrics = zip(*metric_classes)
+
+y_pred = _make_series(n_columns=2, n_timepoints=20, random_state=21)
+y_true = _make_series(n_columns=2, n_timepoints=20, random_state=42)
```

Comment:
Is it worth creating two tests sets, one univariate and one multivariate?

Reply:
yes, good idea

Reply:
added them